py/sentry_streams/user_functions/sample_filter.py

```diff
@@ -0,0 +1,17 @@
+import json
+
+
+class EventsPiplineFilterFunctions:
+    @staticmethod
+    def simple_filter(value: str) -> bool:
+        d = json.loads(value)
+        return True if "name" in d else False
+
+    @staticmethod
+    def wrong_type_filter(value: str) -> str:
+        # TODO: move test functions into the tests/ folder somehow
+        """
+        Filter with wrong return type, used in tests
+        """
+        d = json.loads(value)
+        return "True" if "name" in d else "False"
```

Comment:
If your test is suppsoed to fail here you do not need to provide a body to the function: Just `pass`